docs/shop_administration/product_categories.md

```diff
@@ -0,0 +1,92 @@
+
+1. In the left panel, click **Product catalog** -> **Categories** to navigate to the product category tree.
+2. To add a category by using the **Create** button, in the category tree, select the category to which you want to add a child category. After the details of the selected category are loaded, click the **Create** button and skip to step 4.
+3. To add a category directly in the category tree, click the three dots icon next to the category to which you want to add a child category. In the context menu, click **Add tag**.
+4. In the product category editing modal, make the following settings:
+    * Enter a unique identifier for the product category, for example, "HW_WIN".
+    * Enter a unique name for the product category, "Devices with Windows".
+    * Optionally, change the parent category of the currently edited category: 
+    
+          In the **Parent** area, click the **Change path** button. Then, in the **Select parent** modal, select a category from the tree and click **Select parent**.
+        
+    * Click **Save** to save the product category.
+
+The newly created category can now be assigned to products.
+
+## Modify product category assignments to products
+
+You can create or change product category relationships with products in two ways: either when you edit or create a Product item, or when you edit a product category.
+
+!!! note
+
+    Products must be of Product Type which is [enabled for categorization](#enable-product-categories). 
+
+### Assign product categories by editing product details
+
+1. In the left panel, click **Product catalog** -> **Products** to navigate to the product list.
+2. To create a new Product item, click the  **Create** button, select the product type with product categories enabled, and click **Add**. Then skip to step 4.
+3. To edit an existing Product item, click the **Edit** icon next to a name of the Product item that you want to modify.
+4. In the Product Type editing modal, go to the **Product category** area and click the **Select product categories** button.
+5. In the pop-up window, select checkboxes next to product categories related with the product.
+6. Click **Select product categories** to close the pop-up window.
+7. Optionally, modify other product information as required.
+8. Click **Create** or **Update** to save your changes.
+
+![Adding product categories](img/product_category_selection.png)
+
+### Assign products by editing product category details
+
+1. In the left panel, click **Product catalog** -> **Categories** to navigate to the product category tree.
+2. In the product category tree, select a category and go to the **Content** tab.
+3. To add products to a list of related products:
+    1. Click the **Assign** button.
+    2. In the Content browser, select checkboxes next to products related with the category.
+    3. Click **Confirm** to confirm the selection.
+4. To remove products from a list of related products:
```

Comment:
I'm not sure whether the word "remove" is appropriate here as there is Unassign word on the UI. It can be little bit confusing for a user. It doesn't remove, just unassigns a Content from the category.